### PR TITLE
Clear up definitions in the attributes section.

### PR DESCRIPTION
**Pull request by jede**
Hi!

I tried to make the description of the attributes object a bit more clear. My changes does not intend to change the spec, just make it a bit easier to understand. The namespace wording in my change is just taken from the relationships section.

### CODE REVIEW

File: format/index.md [hunk 1]
```diff
@@ -194,14 +194,17 @@ consistently throughout an implementation.
...
 
-The value of the `"attributes"` key is a JSON object (an "attributes object")
-that represents information about the resource object it is contained within.
+The value of the `"attributes"` key is an "attributes object". The attributes
+object is a JSON object that represents information about the resource object.
 
-The top level of this object shares a namespace with the members of `relationships`
-and **MUST NOT** contain `id` or `type` members. Apart from these restrictions,
-this object can contain members keyed by any string valid for this specification.
+The members of the attributes object ("attributes") shares namespace with the
```
Comment (tkellen) [inline format/index.md]: _share a namespace_

**Comment by tkellen** [comment]
:+1: to merge after that nit.

**Comment by jede** [comment]
Oh, thanks! Fixed it.

**Comment by tkellen** [comment]
Thank you!

**Pull request merged**